### PR TITLE
x11-misc/libinput gestures: version bump and cleanup

### PR DESCRIPTION
**Pull request by SpiderX**

### CODE REVIEW

File: x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
```
Comment (a17r) [inline x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: could you check if it may support `python3_7`?

Comment (SpiderX) [reply x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: I've already tried it. Repoman produces too many errors with python3_7.
I rather avoid use python3_7 until it will be supported in packages in DEPEND.

File: x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+HOMEPAGE="https://github.com/bulletmark/libinput-gestures"
+if [[ ${PV} == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/bulletmark/${PN}.git"
+else
+	SRC_URI="https://github.com/bulletmark/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
```
Comment (mgorny) [inline x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: README says it's GPL-3+

File: x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+SLOT="0"
+IUSE="test"
```
Comment (mgorny) [inline x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: Also `RESTRICT="!test? ( test )"`

File: x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	>=dev-libs/libinput-1.8.0
+	x11-misc/xdotool
+	x11-misc/wmctrl"
+DEPEND=">=dev-libs/libinput-1.8.0
+	dev-util/desktop-file-utils
+	test? ( dev-python/flake8[${PYTHON_USEDEP}] )"
```
Comment (mgorny) [inline x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: Is flake8 really necessary? We'd rather avoid depending on it given it won't find anything new compared to what upstream tests for already.

File: x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild (hunk 2 above)
Comment (SpiderX) [reply x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: Changed.

File: x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild (hunk 3 above)
Comment (SpiderX) [reply x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: Added.

File: x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild (hunk 4 above)
Comment (SpiderX) [reply x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: Upstream may not check some or all release with flake8 in future.

Comment (mgorny) [reply x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: Let me rephrase. We really want you **not** to depend on flake8, as it's not going to gain py3.7 support anytime soon.

Comment (SpiderX) [reply x11-misc/libinput-gestures/libinput-gestures-2.38.ebuild]: That is a really much cleaner rephrasing :) Especially when it has a strict reason. Test phase removed.

**Comment by mgorny** [comment]
Could you test py3.7 now? ;-)

**Comment by mgorny** [comment]
Thanks. The ebuilds look good now. However, I need your sign-off on both commits. I'm sorry that I missed that before.

**Comment by SpiderX** [comment]
I signed-off both.

**Comment by mgorny** [comment]
```
>>> Test phase: x11-misc/libinput-gestures-2.38
make -j12   check
flake8 libinput-gestures
make: flake8: Command not found
make: *** [Makefile:29: check] Error 127
```

Please disarm, e.g. `src_test() { :; }`.

**Comment by mgorny** [comment]
Hmm, or I'll do it to have it merged now.
